tests/data/transcription/_readme/readme.txt

```diff
@@ -0,0 +1,25 @@
+Reference filename mappings:
```

Comment:
Put this in a PR comment, or in a docstring in the transcription test file, or something.  No need to create a separate directory and read.